app/controllers/api/pages_controller.rb

```diff
@@ -58,6 +58,12 @@ def similar
     render :index, format: :json
   end
 
+  def total_donations
+    @page = Page.find(params[:page_id])
+    total_donations = FundingCounter.convert(page: @page, currency: params[:currency])
```

Comment:
Is there a reason why we're not using an after_create on the transactions model? It would seem safer in terms of being seamless and keeping it transactional.